src/planscape/config/boundary.json

```diff
@@ -28,177 +22,38 @@
         {
             "boundary_name": "huc10",
             "display_name": "HUC-10",
-            "vector_name": "southern-california:vector_huc-10",
-            "region_name": "none",
+            "vector_name": ":vector_huc10",
```

Comment:
double checking here - did the vector file for huc-10/huc10 in geoserver get renamed?

Reply:
George just put in the change in dev-geo and after we test it out he'll make the change in geo (prod)

Reply:
confirmed that both huc-10 and huc10 are loaded on dev-geo, which will be the same for prod before the next prod push